src/CattyTests/PenFeatureTests.swift

```diff
@@ -0,0 +1,254 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
```

Comment:
Please move this file to the `Scene` subfolder. Along with this please also move the `SceneTests`.